package.json

```
@@ -1,40 +1,86 @@
 {
   "name": "starter-kit",
-  "version": "0.1.0",
+  "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build": "npm-run-all build:*",
-    "build:scss": "node-sass 'assets/scss' -o 'dist/assets/css' --output-style compressed",
-    "dev": "nodemon -e js,scss --exec \"npm run lint\" --ignore dist",
-    "lint": "npm run eslint && npm run stylelint",
-    "eslint": "eslint 'assets/js/*.js'; exit 0",
-    "stylelint": "stylelint 'assets/scss/**/*.scss'; exit 0",
+    "build": "npm-run-all build:clean build:html build:js",
+    "build:clean": "rm -rf dist/ && rm -rf public/",
```

Because this is an `rm -rf` we should probably be pretty specific about what we're removing. This change will prevent any `dist/` or `public/` directories the user has in `$PATH` from being targeted accidentally.
```suggestion
    "build:clean": "rm -rf ./dist/ && rm -rf ./public/",
```

---

Good catch @tgardnerFusion